packages/samples/mvp-dapp/src/exception.ts

```diff
@@ -0,0 +1,21 @@
+import { Context, HttpError, Next } from 'koa'
+
+export function exceptionHandle() {
+  return async (ctx: Context, next: Next) => {
+    try {
+      await next()
+    } catch (err) {
+      if (err instanceof HttpError) {
```

Comment:
Would it be an `Error` but not a `HttpError`?

Reply:
> Would it be an `Error` but not a `HttpError`?

Users can customize business errors, so here are simply defined three cases, system errors, business errors, HttpError.

Reply:
My fault, I misunderstood the nested branches.